item-search/examples.md

```diff
@@ -50,6 +50,41 @@ Response with `200 OK`:
 
 Following the link `https://stac-api.example.com/search?page=2` will send the user to the next page of results.
 
+#### Simple GET based search showing the last page of results
+
+Request:
+```
+HTTP GET /search?bbox=-110,39.5,-105,40.5
+```
+
+Response with `200 OK`:
+
+```json
+{
+    "type": "FeatureCollection",
+    "features": [],
```

Comment:
If the resultset is empty, the previous page should've been the last page?!

Reply:
updated the example to include [ ... ] to indicate a non-empty result.

Reply:
Thanks.